src/pages/use-cases/kube-proxy.jsx

```diff
@@ -0,0 +1,86 @@
+  imageAlt: 'Electrician Bee',
+};
+
+const sectionContent1 = {
+  title: 'Kubernetes Without kube-proxy',
+  description:
+    'Cilium’s control and data plane has been built from the ground up for large-scale and highly dynamic cloud native environments where 100s and even 1000s of containers are created and destroyed within seconds. Cilium’s control plane is highly optimized, running in Kubernetes clusters of up to 5K nodes and 100K pods. Cilium’s data plane uses eBPF for efficient load-balancing and incremental updates, avoiding the pitfalls of large IPtables rulesets.',
+  videoSrc: 'https://www.youtube.com/embed/bIRwSIwNHC0',
+};
+
+const sectionContent2 = {
+  title: 'Seamless Integration and Upgrade',
+  description:
+    'If you already have kube-proxy running as a DaemonSet, transitioning to Cilium is a breeze. Replacing kube-proxy with Cilium is a straightforward process, as Cilium provides a Kubernetes-native implementation that is fully compatible with the Kubernetes API. Existing Kubernetes applications and configurations can continue to work seamlessly with Cilium. ',
+  imageSrc: KubeProxyImage1,
+  imageAlt: 'kube proxy illustration',
+  whiteBackground: true,
+};
+
+const sectionContent3 = {
+  title: 'Unlock Advanced Configuration Modes',
+  description:
+    "Cilium's kube-proxy replacement offers advanced configuration modes to cater to your specific needs. Features like client source IP preservation ensure that your service connections remain intact, while Maglev Consistent Hashing enhances load balancing and resiliency. With support for Direct Server Return (DSR) and Hybrid DSR/SNAT modes, you can optimize traffic routing and improve performance.  ",
+  imageSrc: KubeProxyImage2,
+  imageAlt: 'kube-proxy with direct server return mode illustration',
+  imageRight: false,
+};
+
+const testimonials = [
+  {
+    logo: <PostFinanceLogo />,
+    title: 'Post Finance picks Isovalent Cilium for Cloud Native Networking',
+    CTAtext: 'Read The Case Study',
+    url: 'https://isovalent.com/data/isovalent-case-study-postfinance.pdf',
+    description:
+      'Cilium helped our team to build a scalable Kubernetes platform which meets our demanding requirements to run mission-critical banking software in production. We were able to solve the scale issues of our previous CNI plugin and to simplify our Kubernetes setup by eliminating the need for kube-proxy.',
+  },
+  {
+    title: 'Service Load-Balancing Without Kube-Proxy on Kubermatic',
+    logo: <KubermaticLogo />,
+    CTAtext: 'Read The Blog Post',
+    url: 'https://www.kubermatic.com/blog/cilium-cni-integration-in-kubermatic-kubernetes-platform/',
+    description:
+      'We use Cilium because it uses eBPF and provides high throughput, lower latency, and less resource consumption, especially when the number of services is large. It also works well across all three major cloud providers, including Google GKE and Azure AKS, which was a critical factor in our choice.',
+  },
```

Comment:
```suggestion
  },
  {
    title: 'Why is the kernel community replacing iptables with BPF?',
    logo: < />,
    CTAtext: 'Read The Blog Post',
    url: 'https://cilium.io/blog/2018/04/17/why-is-the-kernel-community-replacing-iptables/',
    description:
      'Read the history of iptables in the kernel and the advantages of replacing it with eBPF',
  },
```

Reply:
This also needs a logo

Reply:
Is the description text meant for me?

Reply:
@paularah  Todo

Reply:
> Is the description text meant for me?

@xmulligan

Reply:
@paularah yes the description text above is fine, it just needs a logo now